packages/strapi-admin/admin/src/containers/Users/ListPage/Header.js

```diff
@@ -36,7 +36,7 @@ const Header = ({
 
           {
             color: 'primary',
-            icon: <Envelope style={{ verticalAlign: 'middle' }} />,
+            icon: <Plus style={{ verticalAlign: 'middle' }} />,
```

Comment:
Hello! Is there any particular reason behind that change? :)

Reply:
Hi there! If you notice the name of the button is "add user" or "create user". So the icon should be the "Plus" instead of "envelope" to be consistent with the name of the button.

If the envelope icon was did on purpose, let me know I will modify the code. Cheers.

Reply:
Thanks for your quick answer! The envelope icon was indeed made on purpose. Since the way to "Create" and therefore, invite a new user to the admin panel is by sending a link (hence the mail icon). If you don't mind changing, that'd be awesome! :)

Reply:
Thanks for the explanation, and good to know that! I will revert the code then.